pkg/core/native/policy.go

```diff
@@ -143,6 +163,28 @@ func (p *Policy) OnPersist(ic *interop.Context) error {
 	return nil
 }
 
+// Update updates cached Policy values if they've been changed
+func (p *Policy) Update(dao dao.DAO) {
```

Comment:
`UpdateCache` or `SyncCache` would be better here. But we may also do something else in form of `OnPersistEnd`, NEO contract would make use of it too.

Reply:
Replaced it with `OnPersistEnd`.